ci-operator/step-registry/acm/tests/alc/acm-tests-alc-commands.sh

```
@@ -26,6 +26,15 @@ export CYPRESS_BASE_URL
 CYPRESS_OC_CLUSTER_PASS=$(cat $SHARED_DIR/kubeadmin-password)
 export CYPRESS_OC_CLUSTER_PASS
 
+# Playwright equivalents (required by start.sh after Cypress→Playwright migration)
+export HUB_URL=$(oc whoami --show-console)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)